src/layout/ExpandableTable.js

```diff
@@ -4,7 +4,10 @@ import React from 'react'
 import { observable } from 'mobx'
 import { observer, inject } from 'mobx-react'
 
-let ExpandableTable = inject(() => {
+export let Column = _.identity
+Column.displayName = 'Column'
+
+let ExpandableTable = inject((stores, props) => {
```

Comment:
This code could benefit from being slip into at least two, but probably even more, parts. Definitely the inject and the component parts.

Reply:
I split it up some, is it easier to follow this way?